orchestrator-salt/src/main/resources/salt/salt/fluent/init.sls

```diff
@@ -62,9 +62,12 @@ warning_fluentd_os:
 {% endif %}
 install_fluentd_plugins:
   cmd.run:
-    - name: /opt/td-agent/embedded/bin/fluent-gem install fluent-plugin-cloudwatch-logs
+    - names:
+      - /opt/td-agent/embedded/bin/fluent-gem install fluent-plugin-cloudwatch-logs
+      - curl -k -o /tmp/fluent-plugin-databus-1.0.0.gem https://cloudera-service-delivery-cache.s3-us-west-2.amazonaws.com/fluent-plugin-databus-1.0.0.gem
```

Comment:
Can you please consider:

1. omitting `-k`
2. using `s3.amazonaws.com` instead of region-specific location
3. including `-L`?